`cargo build --release` on Android error !

error[E0428]: the name `unix` is defined multiple times
   --> core/io/mod.rs:368:9
    |
360 |         mod unix;
    |         --------- previous definition of the module `unix` here
...
368 |         mod unix;
    |         ^^^^^^^^^ `unix` redefined here
    |
    = note: `unix` must be defined only once in the type namespace of this module

   Compiling option-ext v0.2.0
error[E0252]: the name `UnixIO` is defined multiple times
   --> core/io/mod.rs:370:17
    |
362 |         pub use unix::UnixIO;
    |                 ------------ previous import of the type `UnixIO` here
...
370 |         pub use unix::UnixIO;
    |                 ^^^^^^^^^^^^ `UnixIO` reimported here
    |
    = note: `UnixIO` must be defined only once in the type namespace of this module

error[E0252]: the name `SyscallIO` is defined multiple times
   --> core/io/mod.rs:371:17
    |
364 | ...se PlatformIO as SyscallIO;
    |       ----------------------- previous import of the type `SyscallIO` here
...
371 | ...se unix::UnixIO as SyscallIO;
    |       ^^^^^^^^^^^^^^^^^^^^^^^^^ `SyscallIO` reimported here
    |
    = note: `SyscallIO` must be defined only once in the type namespace of this module

error[E0252]: the name `PlatformIO` is defined multiple times
   --> core/io/mod.rs:372:17
    |
363 | ...se unix::UnixIO as PlatformIO;
    |       -------------------------- previous import of the type `PlatformIO` here
...
372 | ...se unix::UnixIO as PlatformIO;
    |       ^^^^^^^^^^^^^^^^^^^^^^^^^^ `PlatformIO` reimported here
    |
    = note: `PlatformIO` must be defined only once in the type namespace of this module

Regards!

--- a/core/io/mod.rs
+++ b/core/io/mod.rs
@@ -506,15 +506,7 @@ cfg_block! {
         pub use PlatformIO as SyscallIO;
     }
 
-    #[cfg(any(target_os = "android", target_os = "ios"))] {
-        mod unix;
-        #[cfg(feature = "fs")]
-        pub use unix::UnixIO;
-        pub use unix::UnixIO as SyscallIO;
-        pub use unix::UnixIO as PlatformIO;
-    }
-
-     #[cfg(target_os = "windows")] {
+    #[cfg(target_os = "windows")] {
         mod windows;
         pub use windows::WindowsIO as PlatformIO;
          pub use PlatformIO as SyscallIO;

